nucypher/cli/characters/felix.py

```diff
@@ -102,7 +102,7 @@ def felix(click_config,
                                                                   rest_port=port,
                                                                   db_filepath=db_filepath,
                                                                   poa=poa)
-    except FileNotFoundError:
+    except FileNotFoundError as e:
```

Comment:
Are you thinking that the `FileNotFoundError` may be raised by another condition?

Reply:
There are 3 different filepaths referred to in the above method call and I guess I wanted to know which one it was.  I'll remove that.

Reply:
It's a good Idea, if you can improve the accuracy here as bit.